src/dashboard/Settings/components/SettingsLayout/components/SystemSettings/SystemSettings.css

```diff
@@ -0,0 +1,24 @@
+.IncidentTypedescription{ 
```

Comment:
I think it was agreed to name css file as `styles.css'

Reply:
okay I wasn't there during discussion, but i will fix it